src/components/views/rooms/EventTile.js

```diff
@@ -645,6 +645,14 @@ module.exports = withMatrixClient(React.createClass({
         const timestamp = this.props.mxEvent.getTs() ?
             <MessageTimestamp showTwelveHour={this.props.isTwelveHour} ts={this.props.mxEvent.getTs()} /> : null;
 
+        const room = this.props.matrixClient.getRoom(this.props.mxEvent.getRoomId());
+
+        const BridgeError = sdk.getComponent('messages.BridgeError');
```

Comment:
W.r.t. receiving updates, the way to go here is probably similar to how reactions work:
 - `EventTile::_onReactionsCreated` waits for the `Relations` object to be created for this event by a first incoming relation. This assigns `state.reactions` (which should probably become `state.relations`), and causes a re-render of the EventTile.
 - the `Relations` object (as `state.reactions`) is then passed during `render` to your react subcomponent (`BridgeError`), which should probably only be rendered if `state.reactions` is set.
 - Inside `BridgeError` you monitor the `Relations.add` event on the `Relations` object passed in through a prop, and when triggered, check if it was a bridge error.

Reply:
Thanks for giving me this information. I used it as a starting point, went nearly insane and then did it in a totally different way. At least I find it very neat now :)